### PR TITLE
feat: add filterByURIs for all searchables

### LINKED ISSUE #221: Endepunkt for å hente en liste med geonorge-lokasjoner basert på liste med urier

### PR DESCRIPTION
**Pull request by NilsOveTen**
resolve #221

### CODE REVIEW

File: src/main/java/no/fdk/referencedata/geonorge/administrativeenheter/kommune/KommuneService.java [hunk 1]
```diff
@@ -85,6 +86,11 @@ public Stream<SearchHit> search(String query) {
                 .map(Kommune::toSearchHit);
     }
 
+    public Stream<SearchHit> filterByURIs(List<String> uris) {
```
Comment (jeffreiffers) [inline src/main/java/no/fdk/referencedata/geonorge/administrativeenheter/kommune/KommuneService.java:89]: Filter eller find?

File: src/main/resources/graphql/schema.graphqls [hunk 2]
```diff
@@ -313,6 +314,11 @@ input SearchRequest {
     types: [SearchAlternative!]!
 }
 
+input FilterByURIsRequest {
```
Comment (jeffreiffers) [inline src/main/resources/graphql/schema.graphqls:317]: Filter eller find?

File: src/main/java/no/fdk/referencedata/geonorge/administrativeenheter/kommune/KommuneService.java (hunk 1 above)
Comment (NilsOveTen) [reply src/main/java/no/fdk/referencedata/geonorge/administrativeenheter/kommune/KommuneService.java]: 4bea2e10801ed7b26089a00ae3bff813dde7d911

File: src/main/resources/graphql/schema.graphqls (hunk 2 above)
Comment (NilsOveTen) [reply src/main/resources/graphql/schema.graphqls]: 4bea2e10801ed7b26089a00ae3bff813dde7d911

**Review by jeffreiffers — APPROVED** [review]

**Pull request merged**